onnxscript/onnx_opset/_impl/opset1.py

```diff
@@ -0,0 +1,3072 @@
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from typing import Sequence, Tuple, Union
+
+from onnx import GraphProto, TensorProto
+from onnx.defs import get_schema
+
+from ...onnx_types import (
+    BOOL,
+    COMPLEX64,
+    COMPLEX128,
+    DOUBLE,
+    FLOAT,
+    FLOAT16,
+    INT8,
+    INT16,
+    INT32,
+    INT64,
+    STRING,
+    UINT8,
+    UINT16,
+    UINT32,
+    UINT64,
+)
+from ...values import Op, Opset
+
+
+class Opset1(Opset):
+    def __new__(cls):
+        return Opset.__new__(cls, "", 1)
+
+    def __init__(self):
+        super().__init__()
+
+    def Sigmoid(
+        self, X: Union[DOUBLE, FLOAT, FLOAT16], consumed_inputs: Sequence[int] = None
+    ) -> Union[DOUBLE, FLOAT, FLOAT16]:
+        """
+        Sigmoid takes one input data (Tensor<T>) and produces one output data
+        (Tensor<T>) where the sigmoid function, y = 1 / (1 + exp(-x)), is applied to the
+        tensor elementwise.
+
+        :param X: Input tensor
+        :param consumed_inputs: legacy optimization attribute.
+        """
+
+        schema = get_schema("Sigmoid", 1, "")
+        op = Op(self, "Sigmoid", schema)
+        return op(X, consumed_inputs=consumed_inputs)
+
+    def Gemm(
+        self,
+        A: Union[DOUBLE, FLOAT, FLOAT16],
+        B: Union[DOUBLE, FLOAT, FLOAT16],
+        C: Union[DOUBLE, FLOAT, FLOAT16],
+        alpha: float = 1.0,
+        beta: float = 1.0,
+        broadcast: int = 0,
+        transA: int = 0,
+        transB: int = 0,
+    ) -> Union[DOUBLE, FLOAT, FLOAT16]:
+        """
+        General Matrix multiplication:
+        https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms#Level_3
+        Compute Y = alpha * A * B + beta * C, where input tensor A has
+        dimension (M X K), input tensor B has dimension (K X N), input tensor C and
+        output tensor Y have dimension (M X N).
+        If attribute broadcast is non-zero, input tensor C will be broadcasted to match
+        the dimension requirement. A will be transposed before doing the computation
+        if attribute transA is non-zero, same for B and transB.
+
+        :param A: Input tensor A
+        :param B: Input tensor B
+        :param C: Input tensor C, can be inplace.
+        :param alpha: Scalar multiplier for the product of input tensors A * B, the default value is 1.0.
+        :param beta: Scalar multiplier for input tensor C, the default value is 1.0.
+        :param broadcast: Whether C should be broadcasted
+        :param transA: Whether A should be transposed
+        :param transB: Whether B should be transposed
+        """
+
+        schema = get_schema("Gemm", 1, "")
+        op = Op(self, "Gemm", schema)
+        return op(
+            A, B, C, alpha=alpha, beta=beta, broadcast=broadcast, transA=transA, transB=transB
+        )
+
+    def Or(self, A: BOOL, B: BOOL, axis: int = None, broadcast: int = 0) -> BOOL:
```

Comment:
`axis: int = None`, should it be `axis: Optional[int] = None`?

Reply:
Yes, I need to add Optional and variadic support